librosa/display.py

```diff
@@ -977,6 +983,21 @@ def __coord_tempo(n, sr=22050, hop_length=512, **_kwargs):
     return basis * (edges + 0.5) / edges
 
 
+def __coord_fourier_tempo(n, sr=22050, hop_length=512, **_kwargs):
+    '''Fourier tempogram coordinates'''
+
+    n_fft = 2 * (n - 1)
+    # The following code centers the FFT bins at their frequencies
+    # and clips to the non-negative frequency range [0, nyquist]
```

Comment:
nonnegative

Reply:
we use non-negative elsewhere in the code, so I'd rather keep as is.